src/Sarif.Viewer.VisualStudio.UnitTests/Converters/CollectionToVisibilityConverterTests.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Windows;
+
+using FluentAssertions;
+
+using Microsoft.Sarif.Viewer.Converters;
+
+using Xunit;
+
+namespace Microsoft.Sarif.Viewer.VisualStudio.Converters.UnitTests
+{
+    public class CollectionToVisibilityConverterTests
+    {
+        [Fact]
+        public void CollectionToVisibilityConverter_CollectionHasElements()
+        {
+            List<int> testList = new List<int>();
```

Comment:
>List<int> [](http://example.com/codeflow?start=12&length=9)

use var

Reply:
update all the other places where in this file.

Reply:
updated all